solr/CHANGES.txt

```diff
@@ -81,7 +81,9 @@ Upgrade Notes
 * SOLR-13985: Solr's Jetty now binds to localhost network interface by default for better out of the box security.
   Administrators that need Solr exposed more broadly can change the SOLR_JETTY_HOST property in their Solr include
   (solr.in.sh/solr.in.cmd) file. (Jason Gerlowski, David Smiley, Robert Muir)
-  
+
+* SOLR-14147: Solr now runs with the java security manager enabled by default. Administrators that need to run Solr with Hadoop will need to disable this feature by setting SOLR_SECURITY_MANAGER=false as an argument to the start command or in one of the Solr init scripts. Other features in Solr could also break. (Robert Muir, marcussorealheis) 
```

Comment:
i think wording 'in the environment' may work better than 'as an argument to the start command'? From doing development work, I always think of the term "arguments" being strongly associated with program's parameters placed after the command (e.g. argc/argv in C, args[] in java, etc), but doing this won't work with these scripts.